addons/cni-cilium/cilium.yaml

```diff
@@ -119,14 +142,14 @@ data:
   cluster-name: default
   # Unique ID of the cluster. Must be unique across all conneted clusters and
   # in the range of 1 and 255. Only relevant when building a mesh of clusters.
-  cluster-id: ""
+  cluster-id: "0"
```

Comment:
We are using 0 in KKP, but the comment says that it should be in the range of 1 and 255. Should we change this?

https://github.com/kubermatic/kubermatic/blob/5ea2f529409b18b10a8c74b533265a5182aef48e/addons/cilium/cilium_v1.12.yaml#L150

Reply:
I think any value is fine here as long we do not use cluster mesh.
Maybe 0 is actually used to indicate its not a cluster mesh